cdap-standalone/pom.xml

```diff
@@ -132,6 +132,11 @@
       <version>${project.version}</version>
     </dependency>
     <dependency>
+      <groupId>co.cask.cdap</groupId>
```

Comment:
hum... since you have direct reference in StandaloneMain, that's ok for now. But I think it's better not to as CDAP should just based on what inside the `ext/operations` directory and load them. You should move those classes needed by SDK/master out of the `cdap-operational-stats` maven module.